packages/trader/src/AppV2/Components/TakeProfitHistory/take-profit-history.scss

```diff
@@ -0,0 +1,37 @@
+.take-profit-history {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 8px;
```

Comment:
Will it be possible to use quill tokens here?

Reply:
added thanks